src/main/java/com/taskmaster/server/projects/ProjectsMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.taskmaster.server.projects;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProjectsMapper {
+
```

Comment:
Delete this file. You can move this method inside the service or even in the Dto itself